code/css/index.css

```diff
@@ -49,10 +52,14 @@ footer {
 }
 
 .menu li:hover {
-	color: white;
+	background-color: #111;
+}
+
+.active {
```

Comment:
I know but i was thinking about adding an active class or id name to the html file